### DISCUSSION TITLE
乘法器的使用方式選擇

### DISCUSSION THREAD

**Discussion by Yanlin900414** [bol-edu/HLS-SOC-Discussions#100] [Q&A]
因為一開始我在報告中看到的乘法器似乎由DSP去形成了
我也嘗試打上該行 (* use_dsp = "no" *) 來使乘法器使用LUT方式形成
想問問Lab3是否兩種方式都可以呢？
還是說有限定某一種

**Comment by boledulab** [comment]
都可以
